CHANGELOG.rst

```diff
@@ -1,6 +1,16 @@
 CHANGELOG
 =========
 
+1.0.2 (2015-07-13)
+------------------
+
+* Adds a switch: ALDRYN_NEWSBLOG_UPDATE_SEARCH_DATA_ON_SAVE that when set to
+  False, prevents the indexing of Articles on save. This is useful in
```

Comment:
I wouldn't use ```indexing```, instead maybe "prevents article data from being saved into search_data".
The reason is because indexing could be misleading since this has no impact on haystack indexing of articles.